oeps/architectural-decisions/oep-0066-arch-role-based-access-control.rst

```diff
@@ -0,0 +1,88 @@
+==============================================
+OEP-66: OEP RBAC (Role Based Access Control)
+==============================================
+
+.. This OEP template is based on Nygard's Architecture Decision Records.
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-66 <oep-0066-arch-role-based-access-control>`
+   * - Title
+     - RBAC (Role Based Access Control)
+   * - Last Modified
+     - 2023-08-16
+   * - Authors
+     - Hilary Sinkoff <hsinkoff@2u.com>
+   * - Arbiter
+     - Feanil Patel <feanil@axim.org>
+   * - Status
+     - Draft
+   * - Type
+     - Architecture
+   * - Created
+     - 2023-08-16
+   * - Review Period
+     - TBD
+
+*******
+Context
```

Comment:
I was surprised that there was no mention of the edx-rbac library. Is that being described below, but in other language? Is that not being discussed at all? It feels like an RBAC OEP should at least acknowledge this library and explain why it is or is not part of this OEP.